examples/combined_ODNP.py

```diff
@@ -26,28 +26,29 @@
 repetition_us = 1e6
 FIR_rd = 0.5e6
 vd_list = r_[2.1e3,2.1e4,3.3e4,6.1e4,1.2e5,1.8e5,2.4e5,3e5]
-T1_powers_dB = r_[30,32,33,34]
 T1_node_names = ['FIR_%ddBm'%j for j in T1_powers_dB]
 max_power = 4 #W
 power_steps = 14
 threedown = True
 # }}}
 #{{{Power settings
 dB_settings = gen_powerlist(max_power,power_steps+1, three_down=threedown)
+T1_powers_dB = gen_powerlist(max_power, 5, three_down=False)
 print("dB_settings",dB_settings)
 print("correspond to powers in Watts",10**(dB_settings/10.-3))
+print("T1_powers_dB",T1_powers_dB)
+print("correspond to powers in Watts",10**(T1_powers_dB/10.-3))
 myinput = input("Look ok?")
-#}}}
-time_list = [time.time()]
 if myinput.lower().startswith('n'):
     raise ValueError("you said no!!!")
+#}}}
 powers = 1e-3*10**(dB_settings/10.)
-time_list.append(time.time())
```

Comment:
these are not used.

Reply:
I will try to run this however, in previous runs when we were first trying this out, without these the nodenames/data would not save properly (I know that makes zero sense and it may well have been due to something else) but agreed they are unecessary

Reply:
Don't know what you're talking about w/ the nodenames here.  This is just a list of times that is not used.

Reply:
Okay I ran it yesterday without the time list stuff and it works!